src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1944,14 +1944,32 @@
     <Field Name="LengthOrCount" Type="BoundExpression?" SkipInVisitor="true" Null="allow"/>
     <!-- Collection element placeholder, used in IteratorBody. -->
     <Field Name="ElementPlaceholder" Type="BoundValuePlaceholder?" SkipInVisitor="true" Null="allow"/>
-    <!-- Statement executed for each collection element. -->
-    <Field Name="IteratorBody" Type="BoundStatement?" SkipInVisitor="true" Null="allow"/>
+    <!-- Code executed for each collection element. -->
+    <Field Name="IteratorBody" Type="BoundNode?" SkipInVisitor="true" Null="allow"/>
```

Comment:
I didn't follow. Looking at the places where `BoundCollectionExpressionSpreadElement` instances are created, I only see `BoundExpressionStatement` for the iterator body. What else do we store there?

Reply:
This change was left over from an earlier iteration. Reverted.